stdlib/public/core/UnicodeScalarProperties.swift

```diff
@@ -1438,13 +1475,16 @@ extension Unicode {
     /// and programs can treat `digit` and `numeric` equivalently.
     case numeric
 
-    internal init?(rawValue: __swift_stdlib_UNumericType) {
+    internal init(rawValue: UInt8) {
       switch rawValue {
-      case __swift_stdlib_U_NT_NONE: return nil
-      case __swift_stdlib_U_NT_DECIMAL: self = .decimal
-      case __swift_stdlib_U_NT_DIGIT: self = .digit
-      case __swift_stdlib_U_NT_NUMERIC: self = .numeric
-      default: fatalError("Unknown numeric type \(rawValue)")
+      case 0:
+        self = .numeric
+      case 1:
+        self = .digit
+      case 2:
+        self = .decimal
```

Comment:
We have lots of sets of magic numbers that are coordinated with the data files. I do wonder if it makes sense to have a C header with constants in it...